maven/bnd-reporter-maven-plugin/pom.xml

```diff
@@ -60,6 +60,11 @@
 			<groupId>biz.aQute.bnd</groupId>
 			<artifactId>biz.aQute.bnd.reporter</artifactId>
 		</dependency>
+		
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
```

Comment:
This dependency needs to be test scoped. We do not want junit in the compile scope.

Reply:
Perhaps that is already handled by the parent pom's dependencyManagement section? @rotty3000

Reply:
correct!